app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
you can unpack these variables in the easiest way, also use the more descriptive names